### DISCUSSION TITLE
Interop between OTBR?

### DISCUSSION THREAD

**Discussion by olavt** [openthread/openthread#10584] [Q&A]
I have an Apple TV and I have also setup an OTBR on a Raspberry Pi 4 with an RCP. I use this with Matter over Thread devices.

I think I have read somewhere that one should be able to form a Thread network across different OTBR's from different vendors in an easy way.

Is it possible for my Apple TV and the Raspberry Pi OTBR to form a single Thread network to be able for my Matter over Thread devices to join one network and use both OTBR's to route the traffic in the network?

**Comment by jwhui** [comment]
To join Thread devices to an existing Thread network, you'll need to obtain the Thread credentials for the existing Thread network. Apple TV Thread credentials are accessible via the [iOS ThreadNetwork API](https://developer.apple.com/documentation/threadnetwork/). This does require having a mobile app to use those APIs.

To address the above limitation, [Thread will soon release an update](https://www.threadgroup.org/news-events/blog/ID/428/Threads-2024-Enhancements-and-What-Theyll-Mean-to-You) that allows retrieving the Thread credentials directly from any Thread border router over the network.
